entrypoint.R

```diff
@@ -0,0 +1,87 @@
+#!/usr/local/bin/Rscript
+
+dht::greeting()
+
```

Comment:
also, can we add a `dht::check_ram(4)` here?  (assuming that 4 GB should still be necessary, right?)